deploy.go

```diff
@@ -537,3 +516,46 @@ func (d *App) UpdateServiceTags(ctx context.Context, sv *Service, added, updated
 	}
 	return nil
 }
+
+func (d *App) taskDefinitionArnForDeploy(ctx context.Context, sv *Service, opt DeployOption) (string, error) {
+	if opt.Revision > 0 {
+		if opt.LatestTaskDefinition {
+			return "", ErrConflictOptions("revision and latest-task-definition are exclusive")
+		}
+		if !opt.SkipTaskDefinition {
+			return "", fmt.Errorf("revision requires skip-task-definition")
+		}
```

Comment:
```suggestion
```

The `ecspresso run` command allows only the `--revision` flag without `--skip-task-definition`.
Because the `--revision` flag is clear to use the registered task definition in the past.

So, I think this check is not needed.